lib/average_end_of_day_balance/main.py

```diff
@@ -20,12 +22,17 @@ def get_average_end_of_day_balance(
         user_uuid: the Pngme user_uuid for the mobile phone user
         utc_starttime: the datetime for the left-hand-side of the time-window
         utc_endtime: the datetime for the right-hand-side of the time-window
+
     Returns:
         the average end-of-day total balance for a given time window
     """
     institutions = client.institutions.get(user_uuid)
     balance_df_list = []
 
+    """
```

Comment:
can you pull this into the main docstring?
or just do an inline comment `# construct timerange since beginning of time; default /balances endpoint will only return latest balance`.